todo_app/templates/index.html

```diff
@@ -10,9 +10,34 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <table class="table" border="0">
+        <tr>
+          <td><strong>Title</strong></td>
+          {% if request.values.get('sort') == None %}
+          <td><strong><a href="/?sort=2" class="table">Status</a></strong></td>
+          {% elif request.values.get('sort') == '2' %}
+          <td><strong><a href="/?sort=1" class="table">Status</a></strong></td>
+          {% elif request.values.get('sort') == '1' %}
+          <td><strong><a href="/?sort=2" class="table">Status</a></strong></td>
+          {% endif %}
+          <td><strong>Actions</strong></td>
+        </tr>
+        {% for item in items %}
+        <tr>
+          <td>{{ item.title }}</td>
+          <td>{{ item.status }}</td>
+          <td><form id="toggle_item_{{ item.id }}" action="/toggle_status" method="POST"><input type="hidden" name="toggle_item_id" value="{{ item.id }}"><a href="#" onclick="document.getElementById('toggle_item_{{ item.id }}').submit()">Toggle Status</a></form><form id="remove_item_{{ item.id }}" action="/remove_item" method="POST"><input type="hidden" name="remove_id" value="{{ item.id }}"><a href="#" onclick="document.getElementById('remove_item_{{ item.id }}').submit()">Remove Item</a></form></td>
```

Comment:
This should not all be on one line. You should generally avoid lines this long anywhere in your codebase.

HTML mostly ignores white-space so you can write it in a way that matches the structure. That way it's easier to read and modify and git will record changes to it more helpfully. i.e. this should be formatted as:
```
<td>
  <form id="toggle_item_{{ item.id }}" action="/toggle_status" method="POST">
    <input type="hidden" name="toggle_item_id" value="{{ item.id }}">
    <a href="#" onclick="document.getElementById('toggle_item_{{ item.id }}').submit()">Toggle Status</a>
  </form>
  <form id="remove_item_{{ item.id }}" action="/remove_item" method="POST">
    <input type="hidden" name="remove_id" value="{{ item.id }}">
    <a href="#" onclick="document.getElementById('remove_item_{{ item.id }}').submit()">Remove Item</a>
  </form>
</td>
```

The slight caveat is that it actually shrinks whitespace to a single space character which is usually fine but in case it's not, you can work around it (with css or by commenting out the whitespace or putting the closing bracket on the next line)

Also, why are you using anchors (`<a>` tags) instead of submit buttons? If it's for styling reasons, it's still preferable to have an actual submit button and style it to look like a hyperlink. The css library called bootstrap is already added to the project so you can just use `<input type="submit" value="Remove Item" class="btn btn-link p-0"/>` to make a button look like a link ("p-0" removes padding). HTML should be semantic to make it easier to read and for accessibility. Plus this saves you having to write any javascript.